filemonitor/FileMonitor.hpp

```diff
@@ -8,22 +8,25 @@
 #include <chrono>
 #include <thread>
 
+// Macro definition for time period 500
+#define TIMEPERIOD 500
```

Comment:
please don't use macros if possible, as they are not safe.
here you can easily substitute it with:
const int TIMEPERIOD_MS{500};
or even better:
constexpr auto kTimePeriodMs{500};

Reply:
ok, corrected